source/solutions-library/digital-underwriting.txt

```diff
@@ -0,0 +1,245 @@
+.. _arch-center-is-digital-underwriting-machinelearning-solution:
+
+=====================================================
+Automating Digital Underwriting with Machine Learning
+=====================================================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Leverage Machine Learning with real-time data processing and automate
+digital underwriting.
+
+**Use cases:** `Gen AI
+<https://www.mongodb.com/use-cases/artificial-intelligence>`__,
+`Analytics <https://www.mongodb.com/use-cases/analytics>`__
+
+**Industries:** `Insurance
+<https://www.mongodb.com/industries/manufacturing>`__,
+`Financial Services <https://www.mongodb.com/industries/financial-services>`__,
+`Healthcare <https://www.mongodb.com/industries/healthcare>`__
+
+**Products and tools:** `Time Series <https://www.mongodb.com/time-series>`__,
+ `Atlas App Services <https://www.mongodb.com/atlas/app-services>`__,
+ `Atlas Triggers <https://www.mongodb.com/docs/atlas/app-services/triggers/>`__,
+ `Atlas Functions <https://www.mongodb.com/docs/realm/functions/>`__,
+ `Atlas Charts <https://www.mongodb.com/products/charts>`__,
+ `Spark Connector <https://www.mongodb.com/docs/spark-connector/current/>`__
```

Comment:
This indentation is causing a slight formatting issue in staging. These lines should all start at column 1.

```suggestion
`Atlas App Services <https://www.mongodb.com/atlas/app-services>`__,
`Atlas Triggers <https://www.mongodb.com/docs/atlas/app-services/triggers/>`__,
`Atlas Functions <https://www.mongodb.com/docs/realm/functions/>`__,
`Atlas Charts <https://www.mongodb.com/products/charts>`__,
`Spark Connector <https://www.mongodb.com/docs/spark-connector/current/>`__
```